beacon-chain/rpc/eth/beacon/blocks.go

```diff
@@ -56,6 +56,7 @@ func (e *blockIdParseError) Error() string {
 
 // GetWeakSubjectivity computes the starting epoch of the current weak subjectivity period, and then also
 // determines the best block root and state root to use for a Checkpoint Sync starting from that point.
+// DEPRECATED: GetWeakSubjectivity endpoint will no longer be supported
```

Comment:
I need to set the deprecated message at the bottom to conform to linting , requires comments to start with public function name

Reply:
Let's go ahead and say it is no longer supported. We won't rugpull this endpoint, but we are no longer supporting it now.